.pre-commit-config.yaml

```diff
@@ -43,7 +43,11 @@ repos:
     hooks:
     -   id: mypy
 
--   repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.3.1
+-   repo: local
     hooks:
     -   id: pylint
+        language: system
+        name: PyLint
+        files: \.py$
+        entry: python3 -m pylint.__main__
+        args: []
```

Comment:
You can reuse args from
https://github.com/kpi-web-guild/django-girls-blog-schenkh/pull/22/files#diff-82c3b36709926de3a1b27bea0d8258ccR50

except for the init-hook

Reply:
@webknjaz what's the right syntax for args? I get an error message when adding each line with square brackets, w/o them and even when both lines are in brackets.

Reply:
<img width="1310" alt="Screen Shot 2019-07-03 at 16 20 20" src="https://user-images.githubusercontent.com/3303922/60594926-af27c400-9dae-11e9-8a31-4ea990e58f52.png">
<img width="1310" alt="Screen Shot 2019-07-03 at 16 21 03" src="https://user-images.githubusercontent.com/3303922/60594927-af27c400-9dae-11e9-86bb-e9953d7b29ff.png">
<img width="1310" alt="Screen Shot 2019-07-03 at 16 21 13" src="https://user-images.githubusercontent.com/3303922/60594928-af27c400-9dae-11e9-93c0-99fe441e4604.png">

Reply:
square brackets mean inline sequence in YAML, whereas multiline sequence should just have same-indented lines starting with `- `

Reply:
Should I somehow 'escape' curly brackets?

Reply:
I personally prefer multiline lists because this way, when you edit things, diffs are clean additions and removals of lines

Reply:
also, plz post normal text logs instead of screenshots. I cannot search/copy from those so they're useless.

Reply:
I see you use a dumb default terminal emulator app. I strongly recommend switching to Kitty — https://sw.kovidgoyal.net/kitty/

Reply:
> Should I somehow 'escape' curly brackets?

Only if they are at the beginning of a node. But your current problem is that you have invalid YAML.

Reply:
Ok, thanks, got this fixed

> also, plz post normal text logs instead of screenshots. I cannot search/copy from those so they're useless.

Ok

> I see you use a dumb default terminal emulator app. I strongly recommend switching to Kitty — https://sw.kovidgoyal.net/kitty/

I'm using zsh + ohmyzsh as we discussed previously. Should I still make a switch?

Reply:
zsh is a shell, omz is a configuration framework for it.
But Kitty is a terminal emulator which will run the shell you selected. It provides a better user experience. And it will still run zsh, of course. It's not a replacement for zsh, but for that other terminal emulator you use.

Reply:
Also, as an addition to omz I recommend using the spaceship theme, which provides nice context-aware indicators for more than just git — https://github.com/denysdovhan/spaceship-prompt.

Reply:
Kitty does not provide better ux for me, probably cuz it's not optimized for mac. In reality I have a hard time using it cuz it's not responsive and I need to learn all its settings yet again cuz they're in text config only. I'll stick to my terminal app for now.

Reply:
I think you're just not used to dealing with real configs which is what's required when you deploy things to servers that don't have any GUI. You'll get there eventually.